src/test/java/htsjdk/variant/variantcontext/writer/VCFWriterUnitTest.java

```diff
@@ -26,10 +26,13 @@
 package htsjdk.variant.variantcontext.writer;
```

Comment:
You need tests to prove that the `setVcfHeader()` works for all concrete implementations of `VariantContextWriter`